src/cc/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ set_target_properties(bcc-static PROPERTIES OUTPUT_NAME bcc)
 # BPF is still experimental otherwise it should be available
 #llvm_map_components_to_libnames(llvm_libs bpf mcjit irreader passes)
 llvm_map_components_to_libnames(llvm_libs bitwriter bpfcodegen irreader linker
-  mcjit objcarcopts option passes x86codegen)
+  mcjit objcarcopts option passes x86codegen powerpccodegen)
```

Comment:
Yes, it is included, but not linked in. BPFModule() uses InitializeNativeTarget() which translates to LLVMInitializeX86Target on x86 and LLVMInitializePowerPCTarget on powerpc. So, while linking, only the platform-specific library is picked up.

```
[root@x86node lib64]# readelf -s libbcc.so.0.1.8 | grep LLVMInitialize
  2708: 00000000011ae7b0     3 FUNC    LOCAL  DEFAULT   11 _ZZ27LLVMInitializeBPFTar
 20644: 0000000001031080    41 FUNC    LOCAL  DEFAULT   11 LLVMInitializeBPFTarget
 25265: 0000000000df3f70    30 FUNC    LOCAL  DEFAULT   11 LLVMInitializeX86Target
 27952: 00000000011adb10   192 FUNC    LOCAL  DEFAULT   11 LLVMInitializeBPFTargetMC
 28406: 0000000000f04490   185 FUNC    LOCAL  DEFAULT   11 LLVMInitializeX86TargetMC
<snip>
```

```
[root@rhel71le build]# readelf -s /usr/lib64/libbcc.so.0.1.8 | grep LLVMInitia
    29: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializeBPFTarget
   125: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializeBPFTargetMC
   142: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializeBPFAsmPrint
   229: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializeBPFTargetIn
   287: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializePowerPCAsmP
   316: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND LLVMInitializePowerPCTarg
<snip>
```

I could be wrong, but that's my understanding.